torchci/components/metrics/panels/TimeSeriesPanel.tsx

```diff
@@ -13,7 +13,7 @@ import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 dayjs.extend(utc);
 
-export type Granularity = "hour" | "day" | "week" | "month" | "year";
+export type Granularity = "milliseconds" | "hour" | "day" | "week" | "month" | "year";
```

Comment:
lol, `milliseconds` granularity before second and minute. May be also add these two for completeness sake?

Reply:
switched to minute, hopefully this is close enough to commit granularity that anything that actually needs a commit granularity can just use minute